README.md

```diff
@@ -54,9 +56,10 @@ MS_GITHUB_TOKEN=abc123 php run.php update --cms-major=5 --branch=next-minor --dr
 | --dry-run | Do not push to github or create pull-requests |
 | --account | GitHub account to use for creating pull-requests (default: creative-commoners) |
 | --no-delete | Do not delete `_data` and `_modules` directories before running |
+| --unsupported | Only update unsupported modules that were supported in the previous CMS major. Must also use `--branch=default-branch`. Can not be used with `--cms-major` option. |
 | --update-prs | Update existing open PRs instead of creating new PRs |
 
-**Note** that using `--branch=github-default` will only run scripts in the `scripts/default-branch` directory.
+**Note** that using `--branch=github-default` will only run scripts in the `scripts/default-branch` directory, unless the `--unsupported` flag is also used in which case it will use `scripts/cms-unsupported`.
```

Comment:
We should probably throw an exception rather than have one option override the other.

Reply:
This PR has already added a hard fail explaining the options are incompatible, rather than doing an override.

Reply:
--branch=github-default is required when running --unsupported - I explained my thinking behind requiring both in a comment below